com.unity.visualeffectgraph/Editor/Data/VFXDataOutputEvent.cs

```diff
@@ -98,6 +99,13 @@ public string eventName
                 });
             }
 
+            foreach (var context in allMatchingVFXOutputEvent)
+            {
+                var compiledData = contextToCompiledData[context];
+                compiledData.indexInSystemDesc = outBufferDescs.Count;
```

Comment:
Looks like a typo outBufferDescs, should be outSystemDescs no?

Reply:
See https://github.com/Unity-Technologies/Graphics/pull/4457#issuecomment-839768246